build.gradle

```diff
@@ -36,6 +36,7 @@ ext {
     kafka_avro_serializer_version = "6.1.0"
     avro_version = "1.10.1"
     webjars_locator_version = "0.40"
+    lombok_version="1.18.16"
```

Comment:
Methinks this line unnecessary; 99% sure spring-boot bom brings in a lombok version itself.

Reply:
I'll test some more things out...100% sure that IntelliJ couldn't find the lombok to import when I added the first annotation.